selfdrive/ui/qt/offroad/developer_panel.cc

```diff
@@ -112,6 +120,12 @@ void DeveloperPanel::updateToggles(bool _offroad) {
   }
   experimentalLongitudinalToggle->refresh();
 
+  // Handle specific controls visibility for release branches
+  enableGithubRunner->setVisible(!is_release);
+  errorLogBtn->setVisible(!is_release);
+  joystickToggle->setVisible(!is_release);
+  longManeuverToggle->setVisible(!is_release);
```

Comment:
Let's handle the others in another PR.
```suggestion
  // Handle specific controls visibility for release branches
  errorLogBtn->setVisible(!is_release);
```